d2core/d2gui/gui_manager.go

```diff
@@ -20,10 +24,12 @@ type GuiManager struct {
 	loadingAnim   d2interface.Animation
 	cursorVisible bool
 	loading       bool
+
+	logger *d2util.Logger
 }
```

Comment:
can embed without the name

Reply:
I'd like to leave d2gui for now, becouse I belive, It can use d2ui's logger after #861.